include/openmc/string_utils.h

```diff
@@ -15,13 +17,23 @@ std::string to_element(const std::string& name);
 
 void to_lower(std::string& str);
 
-int word_count(std::string const& str);
+int word_count(const std::string& str);
 
 vector<std::string> split(const std::string& in);
 
 bool ends_with(const std::string& value, const std::string& ending);
 
 bool starts_with(const std::string& value, const std::string& beginning);
 
+template<typename T>
+inline std::string concatenate(const T& temps)
```

Comment:
Because this utility function could be used for containers that aren't temperatures, let's change the `temps` variable name and loop variable name to something more generic.

Reply:
Solved.